### PR TITLE
chore: remove cnm code

### PR DESCRIPTION
**Pull request by QxBytes**
**Reason for Change**:

Removes CNM code, related docs, makefile targets, and related components. CNI/CNS behavior should not change as we are not of aware of anyone using the cnm.

Removes ipamclient in the cns folder and related handlers:
- release ip
- reserve ip
- ip address utilization
- get unhealthy ips

Confirmed the cni's cns invoker queries uses functions release and reserve ip, but the actual underlying cns path queried is release/reserve ip **configs**. Reference searched for the release/reserve/ip address utilization/get unhealthy ips paths and cannot find anything

**Issue Fixed**:

#2648 

**Requirements**:

- [X] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [X] relevant PR labels added

**Notes**:

### CODE REVIEW

File: cns/api.go [hunk 1]
```diff
@@ -22,11 +22,7 @@ const (
 	DeleteNetworkPath             = "/network/delete"
```
Comment (tamilmani1989) [inline cns/api.go:22]: #should we remove `createnetworkpath` and `deletenetworkpath` too?

**Review by tamilmani1989 — COMMENTED** [review]
lets make sure we remove all docs related to cnm

File: cns/api.go (hunk 1 above)
Comment (QxBytes) [reply cns/api.go:22]: If you know that they are not being used, we can delete them, but they are not used by CNM so they haven't been removed.

**Review by rbtr — APPROVED** [review]

**Review by tamilmani1989 — APPROVED** [review]

**Pull request merged**